docs/toc.md

```diff
@@ -13,6 +13,7 @@
 ***
 - <h3> Starter projects</h3>
 - <h3> [ - FPS starter project]({{urlRoot}}/projects/fps/overview)</h3>
+- <h3> [ - Blank starter project]({{urlRoot}}/projects/blank/overview)</h3>
 
```

Comment:
Just noticed that these should be proper nouns -adding on my version of the ToC.

Reply:
I think ideally, we should mention this in the welcome page but I know that's a bit sensitive.

Reply:
updated 👍